File: python/01/terceiro.py
#!/usr/bin/env python
# -*- coding: latin1 -*-

soma=0;
soma_par=0;
soma_impar=0;


for x in range(10):
	numero = int(raw_input("Digite um numero: "));

	if x == 0:
		maior=numero;
		menor=numero;

	if numero > maior:
		maior = numero;
	
	if numero < menor:
		menor = numero;

	if (numero % 2) == 0:
		soma_par = soma_par + numero	
	else:
		soma_impar = soma_impar + numero

	soma = soma + numero;

print "O maior numero é: ", maior;
print "O menor numero é: ", menor;
print "A soma dos numeros é: ", soma;
print "Soma dos numeros pares é: ", soma_par;
print "Soma dos numeros impares é: ", soma_impar;	

File: php/01/terceiro.php
<?php

$numeros = array (152,65,39,25,24,10,2,8,90,48);
echo "<h1>Entrada de dados: </h1>";

$soma_total = 0;
$soma_par = 0;
$soma_impar = 0;

for ($i=0;$i<10;$i++){
	echo $numeros[$i]."<br />";

	$soma_total = $soma_total + $numeros[$i];

	if($numeros[$i] % 2 == 0){
		$soma_par = $soma_par + $numeros[$i];
	}else{
		$soma_impar = $soma_impar + $numeros[$i];
	}
}

echo "<h1> Resultados </h1>";
echo "Soma total: ".$soma_total."<br />";
echo "Soma dos pares:  ".$soma_par."<br />";
echo "Soma dos impares: ".$soma_impar;


?>